test/benchmark/common/host_guest_bench_runner.sh

```diff
@@ -48,6 +48,9 @@ run_benchmark() {
 # Use a sleep time of 2 minutes (2m) for the Asterinas VM
 run_benchmark "${ASTERINAS_GUEST_CMD}" "${ASTERINAS_OUTPUT}" "2m"
 
+# Wait for the QEMU process to exit
+wait
```

Comment:
Normally, this benchmark is finished after calling `run_benchmark` for Linux. But it is also reasonable to add it in case of running one more benchmarks sequentially.